doc-content/enterprise-only/springboot/bus-app-security-disabling-proc.adoc

```diff
@@ -0,0 +1,32 @@
+[id='bus-app-security-disabling-proc_{context}']
+
+= Disabling Spring Security in a {PRODUCT} business application
+You can configure Spring Security in a {PRODUCT} business application to provide the security context without authentication.
+//Why would I want to do this?
+
+.Prerequisites
+* You have a {PRODUCT} Spring Boot application.
+
+.Procedure
+. In the directory that contains your {PRODUCT} Spring Boot application, open the `business-application-service/src/main/java/com/company/service/DefaultWebSecurityConfig.java` file in a text editor or IDE.
```

Comment:
If this is the 1st instance of "IDE", please write it out: integrated development environment (IDE)